Styleguides.md

```diff
@@ -0,0 +1,38 @@
+# Styleguide
+The mean of this documentation is to list all the most usefull components that we should use on the application.
+Fell free to see the official documentation at [https://vuetifyjs.com/en/components/api-explorer/](https://vuetifyjs.com/en/components/api-explorer/) to have more details, about all components listed below.
+
+## Most usefull components
+### Grid system
+[API](https://vuetifyjs.com/en/components/grids) for the grid system and the following elements  
+* `v-container`
+* `v-row`
+* `v-col`
+
+### Coloring
+The colors can be applied to mostly all components or html elements.
+Just place them as a class property and the style will be applied `ex: <v-btn class="primary">Test btn</v-btn>`  
+* `primary`: use as primary color on elements like butons in forms
+* `secondary`: use when a second behaviour is possible 
+* `accent`: used to emphasis something important
+* `error`: used to notify the user in case of error
+* `warning`: used to notify the user that an action is dangerous 
+* `info`: notify the user
+* `success`: used to notify the user in case of success action
+
+### Form elements
+If you click on the link of a component you will be redirected to the API related to it
```

Comment:
Not really needed. I think people can see it's a link

Reply:
we never know XD